[2025-04-24T14:57:29.042+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-24T14:57:29.050+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_lab.filter_new_data manual__2025-04-24T14:57:21.032705+00:00 [queued]>
[2025-04-24T14:57:29.054+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_lab.filter_new_data manual__2025-04-24T14:57:21.032705+00:00 [queued]>
[2025-04-24T14:57:29.054+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-24T14:57:29.059+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): filter_new_data> on 2025-04-24 14:57:21.032705+00:00
[2025-04-24T14:57:29.064+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=396) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-24T14:57:29.066+0000] {standard_task_runner.py:72} INFO - Started process 398 to run task
[2025-04-24T14:57:29.064+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '***_lab', 'filter_new_data', 'manual__2025-04-24T14:57:21.032705+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpzn2s4a4u']
[2025-04-24T14:57:29.066+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask filter_new_data
[2025-04-24T14:57:29.087+0000] {task_command.py:467} INFO - Running <TaskInstance: airflow_lab.filter_new_data manual__2025-04-24T14:57:21.032705+00:00 [running]> on host 7ce975cba66d
[2025-04-24T14:57:29.134+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***_lab' AIRFLOW_CTX_TASK_ID='filter_new_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T14:57:21.032705+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T14:57:21.032705+00:00'
[2025-04-24T14:57:29.135+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-24T14:57:29.136+0000] {logging_mixin.py:190} INFO - No API data to filter.
[2025-04-24T14:57:29.136+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-24T14:57:29.141+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-24T14:57:29.141+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=***_lab, task_id=filter_new_data, run_id=manual__2025-04-24T14:57:21.032705+00:00, execution_date=20250424T145721, start_date=20250424T145729, end_date=20250424T145729
[2025-04-24T14:57:29.167+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-24T14:57:29.178+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-24T14:57:29.179+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
